README.rst

```diff
@@ -105,9 +105,13 @@ settings: ``lms/envs/common.py``
    # helps to configure probability of displaying the verification forms. Values in range 0 to 1 are allowed, where 0
    # means never and 1 means always display. Default value is 0.5 i.e. 50% chance of displaying the form.
    SHOW_SKILL_VERIFICATION_PROBABILITY = 0.5
-   # Optionally update topic name for verification event emitted when a user verifies tags for an xblock.
-   EVENT_BUS_XBLOCK_VERIFICATION_TOPIC = "learning-custom-xblock-skill-verfied"
-
+   # Include the XBLOCK_SKILL_VERIFIED signal in `EVENT_BUS_PRODUCER_CONFIG` to push the event to event bus
+   # Note: Include `openedx_events` in `INSTALLED_APPS` setting in the host application
+   EVENT_BUS_PRODUCER_CONFIG = {
+       'org.openedx.learning.xblock.skill.verified.v1': [
+           {'topic': 'learning-custom-xblock-skill-verfied', 'event_key_field': 'xblock_info.usage_key', 'enabled': True},
+       ],
+   }
```

Comment:
Is this the only change we need to ensure that we are listening to `XBLOCK_SKILL_VERIFIED` signal whenever it's fired, and producing it to event bus in the specified topic name?

Reply:
@sameenfatima78 Yes. https://github.com/openedx/openedx-events/pull/249 has more information on how this works.

Reply:
Awesome. Have we also introduced this change in `edx-platform` for the remaining xblock events?

Reply:
We have https://github.com/openedx/edx-platform/pull/33269 open but there is an open ADR for some changes to this configuration here: https://github.com/openedx/openedx-events/pull/272